scripts/packages/VSCodeServer/src/display.jl

```diff
@@ -82,6 +82,33 @@ const DISPLAYABLE_MIMES = [
     "image/gif"
 ]
 
+"""
+    DIAGNOSTIC_MIME = "application/vnd.julia-vscode.diagnostics"
+
+Trace format is:
+```
+[
+    Dict(
+        "msg" => ""::AbstractString,
+        "path" => ""::AbstractString,
```

Comment:
Should we use URIs here from the get go? Identifying notebook locations for example always requires URIs, so just settling on that might be easier?

Reply:
Julia code generally doesn't have access to URIs, so I don't see how that would work out. Unrelated, but I think we need to change the file path passed to notebooks so that it includes cell info or something like that.